src/Components/NotificationList/NotificationList.less

```diff
@@ -55,3 +55,7 @@
     color: #666;
     text-align: center;
 }
+
+.user {
+    width: 16%;
+}
```

Comment:
Please add a new line at the end of the file

Reply:
@mikaelmello Added a new line at the end of file.

Reply:
@sashasushko I've made the required changes! Please review this PR.